scripts/airgap.sh

```diff
@@ -0,0 +1,274 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+# ==================================================================================================
+# TODO:
+#
+# Review (Victor): https://github.com/kubewarden/kwctl/pull/375#discussion_r1048740479
+# current approach doesn't use the GH release files kubewarden-crds-1.2.3_kubewarden-defaults-1.2.8_kubewarden-controller-1.2.8_policylist.txt
+# - it does
+# canonical locations of artifacts
+# - please explain (helm charts vs release?)
+#
+# TODO:
+# fix https://docs.kubewarden.io/operator-manual/airgap/requirements
+# blogpost - https://github.com/kubewarden/kubewarden.io/pull/153
+# fix documentation - https://github.com/kubewarden/docs/pull/150
+# - get image / policy list from pulled charts?
+# ==================================================================================================
+
+
+# Defaults
+IMAGES_TAR="kubewarden-images.tar"
+POLICIES_TAR="kubewarden-policies.tar.gz"
+
+PROG=$(basename "$0")
+USAGE="\
+Usage:  $PROG command [--parameters]
+
+  $PROG list [--cert-manager]
+  $PROG pull -l file.json [--dry]
+  $PROG push -l file.json -r registry[:port] [--insecure] [--dry]
+  $PROG install -l file.json -r registry[:port] [--insecure] [--dry]
+
+Parameters:
+  -l|--list       Json file with list of required dependencies
+  -r|--registry   Private registry where artifacts are pushed. Helm charts are configured for this registry
+  --dry           Do not perform actions, only print commands that would be executed
+  --cert-manager  Include cert-manager dependencies in the generated list
+  --insecure      For pull command skip kwctl certificate verification (an alternative to sources.yaml file)
+                  For install disable policyServer verification of registry
+
+Commands:
+  list     Generate json list of dependencies. You can redirect it to a file
```

Comment:
I'm considering if it isn't better to drop the list command; it is difficult to consider all the options that would go here:  which versions? which values.yaml, that can influence the images being deployed? which charts (currently missing kubewarden-controller as I see it)? from which repos do we pull the charts? are they already mirroring the kubewarden charts in their own repo? do they prefer to do `helm pull` + check signatures beforehand instead?

I would just tell the users to obtain (and verify) the kubewarden helm charts on their own in the docs (they would do `helm pull`).

Once they have the kubewarden charts via `helm pull` or similar, untar them to get the `imagelist.txt` and `policylist.txt`, do `./airgap.sh pull`, `./airgal.sh push`, `./airgap.sh scaffold-install` (see below) to create values.yaml, and then commit the files to git or do `helm install` as normal docs.

This exact same process can be used for Kubewarden upgrades by users, just substitute the last step for `helm upgrade -i`.